### PR TITLE
Upgrade to components, ARM changes for HDInsight, New stop/start all jobs button

### PR DESCRIPTION
**Pull request by rohit489**
1.	DataProcessing – 
a.	Rev dependency packages version

2.	ARM Template – 
a.	Change SF durability to silver. 
b.	Set min TLS for HDInsight to 1.2 . 
c.	Add zookeeper node config for HDInsight

3.	Services –
a.	Add API for start and stop all spark jobs

4.	Website – 
a.	Accessibility: Tweak borderline color for tabs on flow definition page, charts on metric page and flows list table on flows page to achieve minimum 3:1 luminosity contrast
b.	Rev dependency packages version
c.	Add button to ‘start all jobs’ and ‘stop all jobs’ on the jobs page

### CODE REVIEW

File: DeploymentCloud/Deployment.Common/Resources/Templates/SF-Template.json [hunk 1]
```diff
@@ -377,22 +377,22 @@
                                         "StorageAccountKey2": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', variables('supportLogStorageAccountName')),'2015-05-01-preview').key2]"
                                     },
                                     "publisher": "Microsoft.Azure.ServiceFabric",
-                                    "settings": {
-                                        "clusterEndpoint": "[reference(parameters('clusterName')).clusterEndpoint]",
-                                        "nodeTypeRef": "[parameters('vmNodeType0Name')]",
-                                        "dataPath": "D:\\\\SvcFab",
-                                        "durabilityLevel": "Bronze",
```
Comment (carlbrochu) [inline DeploymentCloud/Deployment.Common/Resources/Templates/SF-Template.json:384]: we could make that configurable in a future change.  worth logging an issue on github.

File: DeploymentCloud/Deployment.Common/Resources/Templates/Spark-Template.json [hunk 2]
```diff
@@ -139,6 +139,7 @@
 						}
 					]
 				},
+				"minSupportedTlsVersion": "1.2",
```
Comment (carlbrochu) [inline DeploymentCloud/Deployment.Common/Resources/Templates/Spark-Template.json:142]: do we need this for other resources as well?

**Review by carlbrochu — APPROVED** [review]

File: DeploymentCloud/Deployment.Common/Resources/Templates/SF-Template.json (hunk 1 above)
Comment (rohit489) [reply DeploymentCloud/Deployment.Common/Resources/Templates/SF-Template.json:384]: Logged issue #189

File: DeploymentCloud/Deployment.Common/Resources/Templates/Spark-Template.json (hunk 2 above)
Comment (rohit489) [reply DeploymentCloud/Deployment.Common/Resources/Templates/Spark-Template.json:142]: Not aware of any other resource that need this

Comment (rohit489) [reply DeploymentCloud/Deployment.Common/Resources/Templates/Spark-Template.json:142]: other known resources like the website already is in TLS 1.2

**Review by jozavala-msft — APPROVED** [review]

**Pull request merged**